src/Config/Config.php

```diff
@@ -7,6 +7,12 @@
 
 class Config implements ConfigInterface
 {
+
+    /**
+     * @var string
+     */
+    const DEFAULT_CONF_FILENAME = '.tcms.conf';
```

Comment:
This is just the name of the file. The path to it is build dynamically.

Reply:
I am not seeing the comment. Can you post a link ?

Reply:
The pattern on line 55 is repeated 2 more times in ConfigTest.php. Instead of repeating it you can move the logic here and have the constant be calculated at runtime.

Reply:
In PHP constants are calculated at "compile time", they cannot contain expression like that. $_SERVER['HOME']  will not be set when constants is evaluated.

Reply:
Duh. OK, let's make this a function then.

Reply:
@atodorov ping

Reply:
pong -> https://github.com/kiwitcms/phpunit-plugin/pull/18/files#r765025275